extra/1-radio-stations.js

```diff
@@ -26,6 +35,17 @@
  */
 // `getStations` goes here
 
+
+function getStations(listFrequencies) {
+  return getAllFrequencies().filter(isRadioStation);
```

Comment:
Has the `isRadioStation` function been defined? I wonder why the function is commented out below?

Reply:
Here in addition to the created and commented out function, there were dozens of lines of code that were not needed.
This is one of those cases when I did not carefully read the assignment and tried to create everything myself.